### DISCUSSION TITLE
Why timestmap is highlighted?

### DISCUSSION THREAD

**Discussion by raffaelemancuso** [jarun/nnn#1840] [Q&A]
Why the timestamp of my "Dropbox (Maestral)" folder is highlighted?

![image](https://github.com/jarun/nnn/assets/54762742/75cf32b3-8c43-4b6c-8143-f9be18383453)

**Comment by N-R-K** [comment]
https://github.com/jarun/nnn/discussions/1784#discussioncomment-7853545

**Reply by raffaelemancuso** [reply]
@N-R-K is this written in the documentation somewhere?

**Reply by jarun** [reply]
The expectation is that you will learn what it is after using `nnn` for a while.

**Reply by raffaelemancuso** [reply]
which clearly failed as I didn't.

**Reply by jarun** [reply]
And you are the only user so far who has asked this. Unfortunately we can't document every smallest aspect of the program.

**Reply by raffaelemancuso** [reply]
So wikipedia can document every aspect of human life, but "we" can't document every aspect of this software.

That's probably because you didn't open the wiki to third parties.

**Reply by N-R-K** [reply]
> is this written in the documentation somewhere?

The link I shared is quoting the wiki.

> That's probably because you didn't open the wiki to third parties.

You are free to compile a list of visual hints like https://github.com/jarun/nnn/discussions/1785 asks and post there. Just because people cannot directly push to the wiki doesn't mean they can't contribute to it.
